File: Very Easy/trimmed_averages.py
# https://edabit.com/challenge/3yPvbEjskPuuSSCAC
def trimmed_averages(lst):
    lst=sorted(lst)[1:-1]
    return round(sum(lst)/len(lst))

def trimmed_averages(lst):
    i=max(lst)
    c=min(lst)
    lst.remove(i)
    lst.remove(c)
    return round(sum(lst)/len(lst))

print(trimmed_averages([4, 5, 7, 100]), 6)

File: javascript/Very_Easy/trimmedAverages.js
https://edabit.com/challenge/pDc7MLcGYzhCW6wRf

function trimmedAverages (arr) {
	arr.sort((a,b)=>a-b)
    const sum =Math.round(arr.slice(1,arr.length-1).reduce((total,curr)=> total+curr,0)/(arr.length-2))
    return sum
}

console.log(trimmedAverages([4, 5, 7, 100]), 6)
